tensorflow/contrib/tensor_forest/kernels/stats_ops.cc

```diff
@@ -275,10 +292,26 @@ class ProcessInputOp : public OpKernel {
     // with mutexes.
     std::unordered_map<int, std::unique_ptr<mutex>> locks;
     std::unordered_map<int32, std::vector<int>> leaf_examples;
+    // keep a id of subsampled feature which this leaf will use.
```

Comment:
Keep feature ids that were subsampled for each leaf.

Reply:
I was suggesting to update the comment : )
if u prefer the way it is, then change to "keep AN id  of subsampled featureS which this leaf will use